frontend/src/App.js

```diff
@@ -58,6 +62,31 @@ const theme = createMuiTheme({
 });
 
 function App() {
+  const [isAuthenticated, userHasAuthenticated] = useState(false);
+
+  async function Auth() {
+    try {
+      await ky.get("/api/get-access-token");
+      userHasAuthenticated(true);
+      isAuthenticated(true);
+    } catch (e) {
+      isAuthenticated(false);
+    }
+  }
+
+  async function onLoad() {
+    try {
+      await Auth();
+      userHasAuthenticated(true);
+    } catch (e) {
+      console.log(e);
+    }
+  }
+
+  useEffect(() => {
+    onLoad();
+  });
```

Comment:
If you want to only run this function on mounting component, you need to add an empty dependency array to the hook.

Once you add an empty dependency array the linter will start complaining. You should then move the `onLoad` function definition to the `useEffect` body and add whatever dependencies it requires.